Learning-Tools-Interoperability.md

```diff
@@ -0,0 +1,67 @@
+# Learning Tools Interoperability (LTI)
+
+MarkUs integrates with other Learning Management Systems via the LTI standard.
+Currently MarkUs supports the following LMS platforms:
+
+- Canvas
+
+## For LMS Administrators
+
+### Installing MarkUs on an LMS
+
+Each LMS implements their own LTI integration process.
+Typically, only administrators can add LTI integrations.
+
+### Canvas
+
+To add MarkUs to a Canvas instance, see their page on
+[configuring LTI keys](https://community.canvaslms.com/t5/Admin-Guide/How-do-I-configure-an-LTI-key-for-an-account/ta-p/140).
+
+MarkUs can be added via a configuration URL. The configuration URL is available at  /lti_deployments/get_canvas_config
+
+## For Instructors
+
+Once installed in your course, a 'Launch Markus' page will appear in your
+course's navigation (disabled by default), and needs to be added to the navigation:
+
+![Canvas MarkUs Navigation](images/canvas-markus-nav.png)
+
+If you believe MarkUs should be installed in your course but it does not appear,
+contact your LMS admistrators.
+
+> **NOTE:**
+> The additional navigation item will only be visible to instructors and
+> administrators (not students)
+
+### Associating your LMS Course with your MarkUs course
+
+Once MarkUs is configured with your LMS, an association between your
+Canvas course and your MarkUs course must be made.
+Click 'Launch MarkUs' in your Canvas course. If you are not logged in to MarkUs,
+you will be prompted to do so. Once you are logged in, you will be presented with
+a list of MarkUs courses to choose from. Select the course that matches your Canvas
+course and submit the form. If your course does not appear in the list,
+you may click 'create new course', which will create a new
+course based on the Canvas course information with you as an instructor.
```

Comment:
I thought we were preventing instructors from creating new courses (as determined by policies)